mantidproject/mantid
# Simplify setting of ties for local parameters in Multi-dataset fitting interface.

Fixes #14331.

**To test**
1. Add some data to the interface.
2. Define a function.
3. You should be able to fix local parameters (unticked Global option) or set / remove ties both in the dialog and in-place in the function browser.
